backend/router.ts

```diff
@@ -0,0 +1,82 @@
+import { Router, Request, Response} from "express";
+import {DatabaseFunctions} from "./repository";
+import {createHmac} from "crypto";
+
+export class MainRouter {
+    public readonly router: Router = Router()
+
+    constructor() {
+        this.router = Router()
+        this.createRoutes()
+    }
+
+    private createRoutes() {
+        this.router
+            .get('/orders', this.getOrders)
+            .post('/search', this.searchItem)
+            .put('/items/:uuid', this.updateItem)
+            .delete('/items/:uuid', this.deleteItem)
+            .post('/items/add', this.addItem)
+            .get('/basket', this.getBasket)
+            .get('/items', this.getItems)
+            .post('/checkHash', this.adminLogging)
+            .post('/addOrder', this.addOrder)
+    }
+
+    private async getOrders(req: Request, res: Response) {
+        const answer = await DatabaseFunctions.get_orders()
+        res.json(answer)
+    }
+    private async searchItem(req: Request, res: Response) {
+        const {search, minPrice, maxPrice} = req.body
+        const answer = await DatabaseFunctions.item_finder(search, minPrice, maxPrice)
+        res.json(answer)
+    }
+    private async updateItem(req: Request, res: Response) {
+        await DatabaseFunctions.update_product(req.params.uuid, req.body)

```

Comment:
What if product with entered UUID doesn't exist? Would application return 404 HTTP status code in this scenario? Does your application express different scenarios with appropriate HTTP status codes?